app/routes/index.tsx

```diff
@@ -66,7 +69,10 @@ export default function Index() {
           {t("front:calendarGoTo")}
         </GoToPageBanner>
       </div>
-      <ArticlesPeek />
+      <ArticlesPeek articles={articles} />
+      <GoToPageBanner to={ARTICLES_MAIN_PAGE} navItem="sendou_love">
```

Comment:
I don't think we are going to have a nav icon for this initially so maybe GoToPageBanner should support not having one at all. 🤔

Reply:
Is it okay for that banner element to have no icon? If that's fine I can do the change later today

Reply:
Change with regards to making the `navItem` prop be optional is done for the `GoToPageBanner` component. Please let me know if this looks okay visually:

![image](https://user-images.githubusercontent.com/15804376/199099949-cadd4917-70b3-4860-992c-808b93f7f445.png)

(I will address the spacing-related comment in the other comment you made earlier)